src/core/components/operation-summary.jsx

```diff
@@ -82,22 +85,25 @@ export default class OperationSummary extends PureComponent {
 
           {displayOperationId && (originalOperationId || operationId) ? <span className="opblock-summary-operation-id">{originalOperationId || operationId}</span> : null}
 
-          <svg className="arrow" width="20" height="20" aria-hidden="true" focusable="false">
-            <use href={isShown ? "#large-arrow-up" : "#large-arrow-down"} xlinkHref={isShown ? "#large-arrow-up" : "#large-arrow-down"} />
-          </svg>
+          {isShown ? <LargeArrowUpIcon className="arrow" /> : <LargeArrowDownIcon className="arrow" />}
+          
         </button>
 
         {
           allowAnonymous ? null :
             <AuthorizeOperationBtn
```

Comment:
We can access `authorizeOperationBtn` by accessing it as a container in https://github.com/swagger-api/swagger-ui/pull/9029/files#diff-ba0a208a7fbeb62ea84492d0ad63e865f3df16c88cf25ad0835e9b67985600aeR57.

```js
const AuthorizeOperationBtn = getComponent("authorizeOperationBtn", true)
```

Then you don't need to pass the `getComponent` prop in explicitly.